src/psyclone/transformations.py

```diff
@@ -2950,17 +2951,17 @@ class ACCDataTrans(RegionTrans):
 
     For example:
 
-    >>> from psyclone.parse import parse
+    >>> from psyclone.parse.algorithm import parse
     >>> from psyclone.psyGen import PSyFactory
-    >>> api = "NEMO"
+    >>> api = "nemo"
     >>> filename = "tra_adv.F90"
     >>> ast, invokeInfo = parse(filename, api=api)
     >>> psy = PSyFactory(api).create(invokeInfo)
     >>>
     >>> from psyclone.transformations import ACCDataTrans
     >>> dtrans = ACCDataTrans()
     >>>
-    >>> schedule = psy.invokes.get('invoke_0').schedule
+    >>> schedule = psy.invokes.get('tra_adv').schedule
     >>> # Uncomment the following line to see a text view of the schedule
     >>> # print(schedule.view())
     >>> kernels = schedule.children[0].children[0].children[0:-1]
```

Comment:
As discussed, would you mind fixing the docstring for ACCEnterDataTrans too?

Reply:
I've tried but I don't really have any experience with the GOcean API so I'm not sure what I did makes sense.

Reply:
No, I don't *think* so - if there were no compute constructs being added then that was definitely an oversight!

Reply:
I've just confirmed the directive is indeed empty, I'll attempt a fix by inserting a kernels region around both generated loops. To clear this mess, after you are done reviewing, please do let me rewrite history to avoid the spurious commits.

Reply:
Done.